marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/entity-services/test-data/mappings/CustomerMapping/CustomerMapping-1.mapping.json

```diff
@@ -0,0 +1,15 @@
+{
+  "lang": "zxx",
+  "name": "CustomerMapping",
+  "description": "This is intentionally mapping with invalid source properties",
+  "version": 1,
+  "targetEntityType": "http://marklogic.com/data-hub/example/CustomerType-0.0.1/CustomerType",
+  "properties": {
+    "id": {
+      "sourcedFrom": "invalidProp1"
```

Comment:
These aren't really invalid, they're just missing, right? "invalid" implies to me that some sort of error should be thrown.